src/app/[id]/create/page.tsx

```diff
@@ -55,27 +57,18 @@ export default function Page({params}: { params: { id: string } }) {
           alert('Noe gikk galt ved henting av tittelinformasjon. Kontakt tekst-teamet om problemet vedvarer.');
         }
       });
-  }, [params]);
+  }, [params, router, titleString]);
 
   function showSavedMessage() {
     setSaveMessageIsVisible(true);
     setTimeout(() => setSaveMessageIsVisible(false), 5000);
   }
 
-  function validateBetweenZeroAndFive(value: number) {
-    let error;
-    if (value < 0) {
-      error = 'Tallet kan ikke være negativt';
-    } else if (value > 5) {
-      error = 'Tallet kan ikke være større enn 5';
-    }
-    return error;
-  }
-
   async function submitForm(values: title) {
     const res = await postLocalTitle(values);
     if (res.ok) {
       showSavedMessage();
+      setTimeout(() => router.push(`/${params.id}?title=${titleString}`), 5000);
```

Comment:
Hvorfor timeout?

Reply:
For å vise melding om at den er lagra (i 5 sek), så sende deg videre. Trykker man noe så blir man sendt uansett. Er fort litt UI smak-og-behag, men tenkte det var lurt å vise meldinga siden Mona tidligere sa at det var vanskelig å skjønne at ting var lagra.

Reply:
Makes sense! :smile: